geo/src/algorithm/voronoi_diagram.rs

```diff
@@ -446,22 +475,22 @@ where
     }
 
     let tmp_line = if intersections.contains(&true) {
```

Comment:
`intersections` is being thrown away.  it can be type `bool`.  the loop above (`for line in other_lines {..}`) should `break` as soon as it would put a `true` into `intersections`.